data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/serializers/XnodeContingencySerializer.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.farao_community.farao.data.crac_impl.json.serializers;
+
+import com.farao_community.farao.data.crac_api.NetworkElement;
+import com.farao_community.farao.data.crac_impl.XnodeContingency;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.type.WritableTypeId;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.jsontype.TypeSerializer;
+
+import java.io.IOException;
+
+import static com.farao_community.farao.data.crac_impl.json.JsonSerializationNames.*;
+
+/**
+ * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
+ */
+public class XnodeContingencySerializer extends JsonSerializer<XnodeContingency> {
+    @Override
+    public void serialize(XnodeContingency value, JsonGenerator gen, SerializerProvider serializers) throws IOException {
+        gen.writeStringField(ID, value.getId());
+        gen.writeStringField(NAME, value.getName());
+        gen.writeArrayFieldStart(XNODE_IDS);
+        for (String xnodeId : value.getXnodeIds()) {
+            gen.writeObject(xnodeId);
+        }
+        gen.writeEndArray();
+        if (value.isSynchronized()) {
+            gen.writeArrayFieldStart(NETWORK_ELEMENTS);
+            for (NetworkElement networkElement : value.getNetworkElements()) {
+                gen.writeObject(networkElement.getId());
+            }
+            gen.writeEndArray();
+        }
```

Comment:
I'm not sure it is a good idea to write that. 
Until now, we never wrote the syncrhonized values into the json-crac.

Reply:
Ok I'll change this behaviour. (I noticed that in camarero at the end of the RAO, we reimport the json crac and apply the RAs to the network file, without syncing the 2 in-between. But I guess the network export doesn't work with Xnodes anyways)